docker/containers/zeek/Dockerfile.ubuntu:20.04

```diff
@@ -14,60 +14,76 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-ARG FROM_IMAGE="centos"
-ARG FROM_IMAGE_TAG="8"
+ARG FROM_IMAGE="ubuntu"
+ARG FROM_IMAGE_TAG="20.04"
 
 FROM "${FROM_IMAGE}":"${FROM_IMAGE_TAG}"
 
 ARG ZEEK_VERSION
 ARG LIBRDKAFKA_VERSION
 
-# install epel for screen
-RUN dnf install -y epel-release
-
 # copy in the .screenrc
 COPY .screenrc /root
 
-# install powertools for libpcap-devel
-RUN dnf install -y 'dnf-command(config-manager)' && \
-    dnf config-manager --set-enabled powertools
+RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install \
```

Comment:
There should be tools parity between this and centos.  please add in vim

Reply:
and jq and tree

Reply:
You got it